src/posts/2019-05-08-nested-callbacks.md

```diff
@@ -222,8 +221,7 @@ If you take advantage of the single-argument style with promises, you can tweak
 const makeBurger = () => {
   return getBeef()
     .then(cookBeef)
-    .then(getBuns)
-    .then(putBeefBetweenBuns)
+    .then(cookedBeef => getBuns().then(buns => putBeefBetweenBuns(buns, cookedBeef)))
```

Comment:
I'd prefer to keep this as the original. IMO, not a good practice to have nested `then` calls. In this case I'd return an array `[cookedBeef, buns]`, so I can use it in the next `then`. But there's no need to explain that at this point. 

Can you remove the changes here please?